doc/api/globals.md

```diff
@@ -146,6 +146,20 @@ This variable may appear to be global but is not. See [`__dirname`][].
 
 This variable may appear to be global but is not. See [`__filename`][].
 
+## `atob(data)`
+<!-- YAML
+added: REPLACEME
+-->
+
+Global alias for [`buffer.atob()`][].
+
+## `btoa(data)`
+<!-- YAML
+added: REPLACEME
+-->
+
+Global alias for [`buffer.btoa()`][].
```

Comment:
Let's make sure that these also get the Legacy status introduced in https://github.com/nodejs/node/pull/37784 :)

Reply:
I'll hold off landing this until after #37784 lands.